dali/base/dadfs.cpp

```diff
@@ -191,18 +191,18 @@ static IPropertyTree *getCostPropTree(const char *cluster)
     }
 }
 
-extern da_decl double calcFileAtRestCost(const char * cluster, double sizeGB, double fileAgeDays)
+extern da_decl cost_type calcFileAtRestCost(const char * cluster, double sizeGB, double fileAgeDays)
 {
     Owned<const IPropertyTree> costPT = getCostPropTree(cluster);
 
     if (costPT==nullptr)
         return 0.0;
```

Comment:
should be changed from returning a floating point number to an integer - i.e. return 0;

Reply:
also on line 210 in calcFileAccessCost